src/main/webapp/app/index.tsx

```diff
@@ -16,6 +19,10 @@ setupAxiosInterceptors(() => mobxStores.authStore.clearAuthentication('login.err
 
 loadIcons();
 
+// Initialize Firebase
+export const firebaseApp = initializeFirebase();
+export const firebaseDB = getDatabase(firebaseApp);
```

Comment:
If we do not check whether firebase configs are in place, we assume firebase is required. Then the oncokb-curation should not even be initialized(pod cannot be in ready state)